### PR TITLE
[READY]Removes fire proof from sandals

### PR DESCRIPTION
**Pull request by jughu**
Sandals are not fireproof or acidproof anymore

:cl: jughu
tweak: sandals are not fireproof or acidproof anymore
:add: Magical sandals for the wizard that are still fireproof/acid proof
:tweak: makes the marisa boots acid and fire proof too
/:cl:

### CODE REVIEW

**Comment by Firecage** [comment]
Ummmm, you just made all sandals Noslip.

**Comment by jughu** [comment]
I though these sandals were special because they werw fireproof and shit WHY ARE SANDALS FIREPROOF

**Comment by jughu** [comment]
Ok im changing the bandwagon

**Comment by Firecage** [comment]
Went to make double Sure. This be all sandals matey. Including those from the clothing vendor and those Botany can make. So yah, probably not a good idea.

**Comment by jughu** [comment]
But u will remove fireproof because that makes no sense

**Comment by Firecage** [comment]
Jughu. Yah, Fire proof doesn't make much sense. Though making the sandals which wizards use a child of normal sandals with fire proof would be a good idea. Since I believe it has fire proof since Sandals are needed as part of the outfit for some spells. Same reason you can't acid it.

**Comment by XDTM** [comment]
Sandals are no longer needed for wizards, so don't worry about it.

**Comment by jughu** [comment]
Yeah sandals arent needed anymore altough i want to add a subtype which is slip proof for the wizard for the spell im planning some later the day

**Comment by jughu** [comment]
Oh yeah i could also remove acid then

**Comment by XDTM** [comment]
Why are you noslipping wizards? They're supposed to be vulnerable, if they want noslip they can use galoshes and take the slowdown.

**Comment by jughu** [comment]
I mean its not like they can take advanced magboots for no slowdown magnetic and no slips while on anyways

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 1]
```diff
@@ -35,13 +35,17 @@
...
 /obj/item/clothing/shoes/sandal/marisa
 	desc = "A pair of magic black shoes."
 	name = "magic shoes"
 	icon_state = "black"
-
+	
+/obj/item/clothing/shoes/sandal/magic
+	name = "magical sandals"
+	desc = "A pair of sandals imbued with magic"
+	resistance_flags = FIRE_PROOF |  ACID_PROOF
```
Comment (RemieRichards) [inline code/modules/clothing/shoes/miscellaneous.dm]: need to add to the marisa ones above too.

**Comment by jughu** [comment]
Should be ready to merge

**Pull request merged**